src/server/svr_jobfunc.c

```diff
@@ -5290,6 +5295,60 @@ svr_clean_job_history(struct work_task *pwt)
 		}
 		/* restore the saved next in pjob */
 		pjob = nxpjob;
+
+		/* check if we spent too long hogging the pbs_server process here */
+		end_time = time(NULL);
+		if ((end_time - begin_time) > SVR_CLEAN_JOBHIST_SECS) {
+			/* Apparently the interval between history purges is too long */
+			/* reduce it using factor 0.7                                 */
```

Comment:
Use a multiline job comment style which is similar to our doxygen header style.  Start with /*.  Each subsequent line starts with *.  End with */ on its own line.  Do this for all the multiline comments you have added.

Reply:
Ok.